src/pages/docs/getting-started/setup.mdx

```diff
@@ -14,9 +14,35 @@ Install reachat in your project by following the steps below.
 
 ### 2. **Install Reablocks and Tailwind**
 
-Follow the [installation steps](https://reablocks.dev/docs/getting-started/setup#install-steps) for Reablocks and Tailwind if they are not already included in your project. Then, initialize the `<ThemeProvider />` to allow for theming the chat component.
+Open your terminal, navigate to your project's root directory, and use the reablocks-cli to initialize reablocks in your project:
```

Comment:
We probably want to give some background to why they are installing reablocks for this lol